cli/cli/Options/SaveToFileOption.cs

```diff
@@ -18,56 +18,53 @@ public static void Bind<T>(AppCommand<T> command)
 	{
 		const string saveToFileStr = "--save-to-file";
 		const string noTokenSaveStr = "--no-token-save";
-
-		const bool saveToFileDefault = true;
-		const bool noTokenSaveDefault = false;
 		
 		// included for legacy reasons. 
 		var saveToFileOption = new Option<bool>(saveToFileStr, 
 			
-			// Hey You, Watch out! This default value affects the algorithm in the parse handling! 
-			getDefaultValue: () => saveToFileDefault, 
+			// This default value is assumed "true" in the SaveToFileOption.ShouldSaveToFile function

```

Comment:
For what it is worth, this wording is _much_ nicer 😄; it is very explicit about what the assumption is and where the assumption matters. 👍